lib/defaults-utils.js

```diff
@@ -46,15 +46,15 @@ export function toXmlArg (value, serialize = true) {
   } else if (_.isString(value)) {
     xmlDoc = new DOMParser().parseFromString(`<string></string>`, 'text/xml');
     const valueTextEl = xmlDoc.createTextNode(value);
-    xmlDoc.documentElement.appendChild(valueTextEl);
+    xmlDoc.documentElement?.appendChild(valueTextEl);
   }
 
-  if (!xmlDoc) {
+  if (!xmlDoc || !xmlDoc.documentElement) {
     throw new TypeError(`The defaults value ${JSON.stringify(value)} cannot be written, ` +
       `because it is not known how to handle its type`);
   }
 
-  return serialize
+  return serialize && xmlDoc.documentElement
```

Comment:
`serializeToString` does not accept null

Reply:
I assume typescript is smart enough to consider the above condition